yudiencore/core.go

```diff
@@ -27,8 +27,18 @@ const (
 	type_map          = iota // map[string]interface{}
 )
 
+const ( // order matters for log levels
+	log_off   = iota
```

Comment:
If you do make as struct (such as log_levels) then you can remove the log_* names since it will be implicit